docs/src/test/kotlin/docs/user/gettingstarted/build.gradle.kts

```diff
@@ -0,0 +1,48 @@
+// #build-kts
+import com.google.protobuf.gradle.*
+
+plugins {
+    kotlin("jvm") version "1.3.72"
+    id("com.google.protobuf") version "0.8.12"
+    id("com.google.cloud.tools.jib") version "2.2.0"
+    idea
+}
+
+repositories {
+    mavenLocal()
+    mavenCentral()
+}
+
+dependencies {
+    implementation(kotlin("stdlib-jdk8"))
+    implementation("io.cloudstate:cloudstate-kotlin-support:$cloudstate.kotlin-support.version$")
```

Comment:
Okay, see that the gradle build has a paradox variable. That won't be substituted.